packages/cli/src/lib/utils/record.ts

```diff
@@ -0,0 +1,5 @@
+type Recordable<T> = {
```

Comment:
Little hack so we can use `CliUx.ux.table` with custom objects array that are not of type `Record<string, ...>`

Reply:
Mh, is it a good idea tho?
Should we not convert our custom stuffs into what the `CliUx.ux.table` into the format it expect?

Reply:
Indeed, instead of a function that simply hack the typing, make this a function that accept an object and transform to an actual `Record<string, ...>`.

With something like: `Object.entries(...).map(...)`